### PR TITLE
New package: font-fantasque-sans-mono-20170605.0

### PR DESCRIPTION
**Pull request by 0x7cc**
A programming font: [fantasque-sans-mono](https://fontlibrary.org/en/font/fantasque-sans-mono).

### CODE REVIEW

File: srcpkgs/font-fantasque-sans-mono/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'font-fantasque-sans-mono'
+pkgname=font-fantasque-sans-mono
+version=20170605.0
```
Comment (the-maldridge) [inline srcpkgs/font-fantasque-sans-mono/template]: The latest version is 1.7.1 I'm not sure where you got this version from.

File: srcpkgs/font-fantasque-sans-mono/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+revision=1
+short_desc="A font family with a great monospaced variant for programmers"
+maintainer="Lenilson Jose Dias <lenilson@gmail.com>"
+license="OFL1.1"
+create_wrksrc=yes
+noarch="yes"
+depends="fontconfig font-util"
+homepage="https://fontlibrary.org/en/font/fantasque-sans-mono"
+font_dirs="/usr/share/fonts/TTF"
+_distrelease=v1.7.1
```
Comment (the-maldridge) [inline srcpkgs/font-fantasque-sans-mono/template]: This should be the package version.

File: srcpkgs/font-fantasque-sans-mono/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+distfiles="https://github.com/belluzj/fantasque-sans/releases/download/${_distrelease}/FantasqueSansMono.tar.gz"
```
Comment (the-maldridge) [inline srcpkgs/font-fantasque-sans-mono/template]: Use ${version} here.

File: srcpkgs/font-fantasque-sans-mono/template [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'font-fantasque-sans-mono'
+pkgname=font-fantasque-sans-mono
+version=20170605.0
+revision=1
+short_desc="A font family with a great monospaced variant for programmers"
+maintainer="Lenilson Jose Dias <lenilson@gmail.com>"
+license="OFL1.1"
+create_wrksrc=yes
+noarch="yes"
+depends="fontconfig font-util"
```
Comment (the-maldridge) [inline srcpkgs/font-fantasque-sans-mono/template]: need to also have xbps-triggers, and makedepends="xbps-triggers"

**Comment by the-maldridge** [comment]
Not really sure what happened here.  This should be one commit, and the version should have been 1.7.1.  Can you expand on why you made the changes you made here?

**Comment by 0x7cc** [comment]
I'm sorry, I made some mistakes about versioning/disfiles... 
There are two commits because of that. The second one was a `push -f`(my bad). 

I expect this third one fix that: #6608 99c6dc7

Thank you for helping me.

**Pull request closed without merging**